### PR TITLE
feat(zenko-367): Add Backbeat CRR tests to eve

### PR DESCRIPTION
**Pull request by tmacro**
This leaves GCP tests skipped until they are fixed

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -153,19 +174,41 @@ stages:
...
     - ShellCommand: *helm_build_dep
-    - ShellCommand: *helm_lint
     - ShellCommandWithSecrets:
         name: Install Zenko !
         command: >-
           helm upgrade $ZENKO_HELM_RELEASE --namespace %(prop:testNamespace)s
           --install charts/zenko
           -f eve/ci-values.yml
-          --timeout 800
+          --timeout 700
```
Comment (jonathan-gramain) [inline eve/main.yml]: interesting change! :)

**Review by jonathan-gramain — APPROVED** [review]

File: eve/main.yml (hunk 1 above)
Comment (giacope) [reply eve/main.yml]: @tmacro There is another timeout with the same value you can also change, for the nightly build stage.

**Review by giacope — APPROVED** [review]

**Comment by tcarmet** [comment]
Just a little note, we might have fixed your gcp key issue, try it out, and if it still fails let us know.

**Review by rahulreddy — APPROVED** [review]
Sylvian's comments on the hipchat room may help with the GCP credentials issue.

**Review by ssalaues — APPROVED** [review]

**Pull request merged**